test/Input.test.jsx

```diff
@@ -0,0 +1,52 @@
+import { fireEvent, render } from '@testing-library/react';
+
+import Input from '../src/Input';
+
+describe('Input', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  const handleChange = jest.fn();
+  const handleClick = jest.fn();
+
+  const renderComponent = (value = '') => {
+    const { container, getByRole } = render((
+      <Input
+        value={value}
+        onChange={handleChange}
+        onClick={handleClick}
+      />
+    ));
+    const input = getByRole('textbox', { name: /할 일/i });
+    const addButton = getByRole('button', { name: /추가/i });
+
+    return {
+      container, getByRole, input, addButton,
+    };
+  };
```

Comment:
```suggestion
  const renderApp = (value = '') => {
    return render((
      <Input
        value={value}
        onChange={handleChange}
        onClick={handleClick}
      />
    ));
  };
```

로 render 역할만 명확히해주고 내부 요소는 나머진 각 독립된 테스트에서 드러내주면 좋을 것 같아요.
지금은 render 함수가 복잡하게 느껴지는 것 같아요.